package.json

```diff
@@ -1,64 +1,79 @@
 {
-  "name": "singularity-directory-webapp-react",
+  "name": "react-next-boilerplate",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "echo 'TODO: we need to add some tests on this project as soon as possible!'",
-    "eslint-staged": "files=`git diff --staged --diff-filter=AM --name-only | grep .js$ | tr '\n' ' '`; if [ -n \"$files\" ]; then eslint $files; fi;",
-    "eslint": "eslint .",
     "dev": "node server.js",
     "build": "next build src",
     "start": "NODE_ENV=production node server.js",
-    "storybook": "start-storybook -p 9009 -c storybook",
-    "precommit": "npm run eslint-staged",
-    "prepush": "npm run eslint && npm run test"
+    "storybook": "start-storybook -p 6006",
+    "eslint": "eslint ./src",
+    "stylelint": "stylelint ./src/**/*.css",
+    "lint": "yarn eslint && yarn stylelint",
+    "build-storybook": "build-storybook"
   },
-  "author": "Daniel Leite de Oliveira <dleitee@gmail.com> (https://github.com/dleitee)",
+  "husky": {
+    "hooks": {
+      "pre-push": "yarn lint"
+    }
+  },
+  "keywords": [],
+  "author": "",
   "license": "ISC",
   "dependencies": {
-    "@zeit/next-css": "^0.2.0",
+    "@zeit/next-css": "^1.0.2-canary.0",
```

Comment:
Isn't a good practice to use versions `beta`, `alpha`, always try the version `stable` of the packages releases.